### PR TITLE
Penv 122 remove goplugins

### PR DESCRIPTION
**Pull request by megge-dream**
**- What I did**

- Remove goplugins, insgorund and api method for upload contracts.
- Comment func tests - they will be rewritten later.
- Remove `builtin` param from LR config - we always use builtin contracts, no need to set specific param for it.

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,8 +16,7 @@ RUN BUILD_NUMBER=${BUILD_NUMBER} \
     BUILD_DATE=${BUILD_DATE} \
     BUILD_TIME=${BUILD_TIME} \
     BUILD_HASH=${BUILD_HASH} \
-    BUILD_VERSION=${BUILD_VERSION} \
-    make insgorund build
+    BUILD_VERSION=${BUILD_VERSION}
```
Comment (Tsovak) [inline Dockerfile]: actually, after deletion make command we don't need this argument. 
I suppose to you deleted extra `make build `stage

File: applicationbase/functest/pressure_test.go [hunk 2]
```diff
@@ -7,239 +7,239 @@
...
-			require.Equal(t, float64(1), bal)
-			time.Sleep(1 * time.Second)
-		}
-	})
-}
+// import (
+// 	"sync"
+// 	"sync/atomic"
+// 	"testing"
+// 	"time"
```
Comment (Tsovak) [inline applicationbase/functest/pressure_test.go]: did we comment it to rewrite/fix latter?

File: applicationbase/functest/test_utils.go [hunk 3]
```diff
@@ -81,143 +77,3 @@ func createMember(t *testing.T) *CommonUser {
 	member.Ref = ref
 	return member
 }
-
```
Comment (Tsovak) [inline applicationbase/functest/test_utils.go]: after deletion I thing `var contracts = map[string]*contractInfo{}` no more need and can be deleted

File: applicationbase/functest/pressure_test.go (hunk 2 above)
Comment (megge-dream) [reply applicationbase/functest/pressure_test.go]: yes, we will rewrite it!

File: applicationbase/functest/test_utils.go (hunk 3 above)
Comment (megge-dream) [reply applicationbase/functest/test_utils.go]: you are absolutely right!

File: Dockerfile (hunk 1 above)
Comment (megge-dream) [reply Dockerfile]: changed it

File: cmd/insgocc/insgocc.go [hunk 4]
```diff
@@ -77,8 +77,6 @@ func (r *machineTypeFlag) Set(arg string) error {
 		fallthrough
 	case "go":
 		fallthrough
-	case "golang":
```
Comment (Vany) [inline cmd/insgocc/insgocc.go]: "go" will fall through too

File: api/main.go [hunk 5]
```diff
@@ -12,7 +12,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/insolar/x-crypto"
+	crypto "github.com/insolar/x-crypto"
```
[the remark this answers is not in the document]
Comment (megge-dream) [reply api/main.go]: without alias `go import` changes it to `crypto` lib

**Pull request merged**